src/users/users.ts

```diff
@@ -217,4 +220,16 @@ export class Users {
 
     return deserializeUser(data);
   }
+
+  async updateUserPassword(payload: UpdateUserPasswordOptions): Promise<User> {
+    const { data } = await this.workos.put<
+      UserResponse,
+      SerializedUpdateUserPasswordOptions
```

Comment:
Based on the other usage, it looks like we might be able to infer this second type parameter:

```suggestion
```